frontend/pages/reports/index.js

```diff
@@ -0,0 +1,227 @@
+import s from 'styled-components'
+import Router from 'next/router'
+import { useEffect, useState } from 'react'
+import renderPage from '../../renderPage.js'
+import { Icon, Empty, Checkbox, CheckboxLabel } from '../../components/common'
+import { doApiRequest, API_BASE_URL } from '../../utils'
+import { Container } from '../../components/common/Container'
+import { CLUBS_GREY } from '../../constants/colors'
+import Edit from './edit'
+
+const GroupLabel = s.h4`
+  margin-bottom: 0em !important;
+  font-size: 32px;
+  color: #626572;
+`
+
+const ColoredHeader = s.div`
+  background: linear-gradient(to right, #ef4c5f, #4954f4);
+  height: 7em;
+`
+
+const TransparentButton = s.button`
+  width: 190px;
+  height: 35px;
```

Comment:
What's the significance of these? For any seemingly arbitrary measurements first (1) make sure they work on a few screen sizes/resolutions/browsers and (2) add a comment above explaining why you chose those values. Else it's hard for other ppl to pick up where you left off. Generally fixing button width instead of adding padding is not good–what if the font size changes or the font changes?